### PR TITLE
fix(workspace): added no issue found text

### LINKED ISSUE #4916: When an issue is not found, display a message that explains what's going on
Right now if I go to a URL of a missing issue (possibly deleted earlier), all I can see is a blank screen:

Only if I open up the console, I can see that Huly couldn’t find the issue.

Instead of displaying a blank screen, display a clear message that tells what happened and offers a workaround, such as opening the list of issues in the current project. Here’s an example:

> 
> 404 (:
> We couldn’t find this issue.
> 
> The issue could have been deleted, or you don’t have sufficient permissions to view it.
> 
> Would you like to [see all issues in this project](insert a link to all issues in the containing project)?

<sub>View in Huly <a href="https://front.hc.engineering/guest/platform?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsaW5rSWQiOiI2NWVkZTYzYTgxN2YxMWQ2ODM3ODQ1OWUiLCJndWVzdCI6InRydWUiLCJlbWFpbCI6IiNndWVzdEBoYy5lbmdpbmVlcmluZyIsIndvcmtzcGFjZSI6InBsYXRmb3JtIiwicHJvZHVjdElkIjoiIn0.KFXO2NcxParJpQwMMWfwtAuHNULwesPr0AsqP-szmQQ">UBERF-5435</a></sub>

### PR DESCRIPTION
**Pull request by aeswibon**
# Contribution checklist

## Brief description

* Adds `IssueNotFound` text when the user goes to an unknown issue URL

## Checklist

* [ ] - UI test added to added/changed functionality?
* [ ] - Screenshot is added to PR if applicable.
* [ ] - Does a local formatting is applied (rush format)
* [x] - Does a local svelte-check is applied (rush svelte-check)
* [ ] - Does a local UI tests are executed [UI Testing](../tests/readme.md)
* [ ] - Does the code work? Check whether the function and logic are correct.
* [ ] - Does Changelog.md is updated with changes?
* [x] - Does the translations are up to date?
* [ ] - Does it well tested?
* [x] - Tested for Chrome.
* [ ] - Tested for Safari.
* [ ] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [x] - Rebase your branch onto the master and upstream branch
* [x] - Is there any redundant or duplicate code?
* [x] - Are required links are linked to PR?
* [ ] - Does the new code is well documented?

## Related issues

* Fixes #4916

## Contributor requirements

* [x] - Sign-off is provided. [DCO](https://github.com/apps/dco)
* [x] - GPG Signature is provided. [GPG](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits)

<sub>View in Huly <a href="https://front.hc.engineering/guest/platform?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsaW5rSWQiOiI2NjBhMDFkZGUwYzE0NzA5ZWRjNWRiODEiLCJndWVzdCI6InRydWUiLCJlbWFpbCI6IiNndWVzdEBoYy5lbmdpbmVlcmluZyIsIndvcmtzcGFjZSI6InBsYXRmb3JtIiwicHJvZHVjdElkIjoiIn0.m8tEQaPqbULWTZTz1dxVEfoAaxpjUNZwaKHo_-R3IMc">UBERF-6241</a></sub>

### CODE REVIEW

File: plugins/workbench-resources/src/components/Workbench.svelte [hunk 1]
```diff
@@ -300,6 +302,9 @@
     if (loc.path.length > 3 && getSpecialComponent(loc.path[3]) === undefined) {
       // resolve short links
       const resolvedLoc = await resolveShortLink(loc)
+      if(locationResolver === "tracker:resolver:Location" && resolvedLoc === undefined) {
```
Comment (haiodo) [inline plugins/workbench-resources/src/components/Workbench.svelte:305]: Not good, workbench should not know about tracker at all.

File: plugins/workbench-assets/lang/en.json [hunk 2]
```diff
@@ -30,6 +30,7 @@
     "PleaseUpdate": "Please update",
     "ServerUnderMaintenance": "Server is under maintenance",
     "MobileNotSupported": "Sorry, mobile devices support coming soon. In the meantime, please use Desktop",
-    "LogInAnyway": "Log in anyway"
+    "LogInAnyway": "Log in anyway",
+    "IssueNotFound": "We couldn't find the issue. The issue may be deleted or you don't have sufficient permissions to view it"
```
Comment (haiodo) [inline plugins/workbench-assets/lang/en.json:34]: Workbench should not know about this kind of document, if we want some notification we need to extend resolvers API to provide necessary labels.

Comment (aeswibon) [reply plugins/workbench-assets/lang/en.json:34]: Okay will update that.

File: plugins/workbench-resources/src/components/Workbench.svelte (hunk 1 above)
Comment (aeswibon) [reply plugins/workbench-resources/src/components/Workbench.svelte:305]: Okay, will try to update the condition in `tracker-resources`

Comment (aeswibon) [reply plugins/workbench-resources/src/components/Workbench.svelte:305]: Kind of stuck. Currently, if we move to an undefined issue URL, so by default, `SpaceView` would be rendered as rest are undefined. And, in `SpaceView`, no content would be rendered as the stuff that it needs are undefined.

**Comment by aeswibon** [comment]
Will open a new PR

**Pull request closed without merging**